cassandane/tiny-tests/JMAPCalendars/calendarevent-participantreply-simple

```diff
@@ -160,8 +160,33 @@ sub test_calendarevent_participantreply_simple
     $self->assert($ical =~ "SEQUENCE:1");
     $self->assert($ical =~ "PARTSTAT=ACCEPTED");
 
+    my ($maj, $min) = Cassandane::Instance->get_version();
+    if ($maj > 3 || ($maj == 3 && $min >= 9)) {
+        xlog $self, "actually update attendee status on the event";
+        $res = $jmap->CallMethods([['CalendarEvent/set', {
+            update => {
+                $id => { "participants/att1/participationStatus" => "accepted" }
+            }
+        }, "R1"]]);
+
+        xlog $self, "set attendee status again";
+        $res = $jmap->CallMethods([['CalendarEvent/participantReply', {
+            eventId => $id,
+            participantEmail => "bugs\@looneytunes.com",
+            updates => {
+                participationStatus => "accepted"
+            }
+        }, "R2"]]);
+        $self->assert_str_equals("2.0", $res->[0][1]{scheduleStatus});
+
+        xlog $self, "verify that NO iMIP messages are sent to organizer/attendee";
+        $data = $self->{instance}->getnotify();
+        (@imips) = grep { $_->{METHOD} eq 'imip' } @$data;
```

Comment:
The parentheses around `@imips` are redundant here AFAIK. It's testing OK without them.

Reply:
Yeah, not sure where that came from.  Fixed in all 3 test files.